### PR TITLE
Feature/assignment 7

### PR DESCRIPTION
**Pull request by mcruz412**
@GabePires read for review! 
The assignment was to print out each car nicely with a message.
I updated the code, esp with the vehicles list and added the switch statement

### CODE REVIEW

File: functionsForFunSwift.playground/Contents.swift [hunk 1]
```diff
@@ -17,8 +17,35 @@ enum DriveTrain {
 struct Vehicle {
     let information: VehicleInformation
     let driveTrain: DriveTrain
+    
+    func vehicleOpinion() {
+        switch driveTrain {
+        case .awd: print("Ohhh, you have a \(information.Model). That's an \(DriveTrain.awd.rawValue), so you can go off-roading with it!")
+        case .rwd: print("If you want fast car, a \(DriveTrain.rwd.rawValue) is what you want. So you'll have fun with your \(information.Model) on the road.")
```
Comment (GabePires) [inline functionsForFunSwift.playground/Contents.swift]: I just realized that the properties in VehicleInformation should begin with a lower case!

Comment (GabePires) [inline functionsForFunSwift.playground/Contents.swift]: Nice touch with the strings!

I would suggest that vehicleOpinion() returns a string though and you do the print on line 47 instead.

**Comment by mcruz412** [comment]
@GabePires I made the following changes per your comment
Question: What's the benefit of returning a string for the the function and doing the print at the for loop? Is that an industry standard?

**Comment by GabePires** [comment]
> What's the benefit of returning a string for the the function and doing the print at the for loop? Is that an industry standard?

Hmm well, previously, in the function, it was kind of doing two things
1. composing a string
2. printing the string
The function itself was not named in such a way that it described the fact it did those two things.
The name of the function mainly just described 1.
If the name of the function was called "func printVehicleOpinion()", then I think that would be a little better.
One other reason for splitting it up is that I think it's good practice if a function does "as little as possible".

**Review by GabePires — APPROVED** [review]
looking good! there's a conflict though.... remember how to fix those? :D

**Comment by mcruz412** [comment]
@GabePires resolved! squashing and merging now

**Pull request merged**